### ISSUE TITLE
Where are the LFE communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/lfe#70]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is LFE supported by one or more large organizations?
* Does LFE have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about LFE? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for LFE? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in LFE? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in LFE?
* Are there popular newsletters for LFE?
* Is LFE taught at programming bootcamps? (If so, what are some examples?)
* Is LFE taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about LFE?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by FranklinChen** [comment]
Has this track been abandoned? I noticed when downloading exercises that they are using very ancient versions of tooling (including the version of lfe itself).

**Comment by yurrriq** [comment]
Hi, I helped start this track and have since gotten very busy with my day job. So, at least I have abandoned it, unfortunately. I know some of the regular Erlang track contributors have done a lot here. Hopefully one of them or someone else can revive it.
